build-tools/packages/build-cli/src/commands/generate/changeset.ts

```diff
@@ -275,6 +292,22 @@ export default class GenerateChangesetCommand extends BaseCommand<
 					}
 				},
 			} as any, // Typed as any because the typings don't include the optionsPerPage property.
+			context.rootFluidBuildConfig.releaseNotes === undefined
+				? undefined
+				: {
+						name: "section",
+						type: "select",
+						choices: sectionChoices,
+						instructions: INSTRUCTIONS,
+						message: "What type of change is this?",
```

Comment:
Hmmm, maybe we should just ask which section instead. I was trying to be a bit vague because I don't think the changeset
metadata should map 1:1 to release notes stuff, and I imagine the metadata growing a bit over time/ being used for more.
But wqe don't have actual plans for that, so asking "what section" is a better question for now.